Replacing opcode images by Wavedrom snippets

nibrunieAtSi5 opened the pull request.

wmat commented:
Can you name the new images with a .edn extension to align with the way it's done in riscv-isa-manual? These aren't really adoc files.

nibrunieAtSi5 commented:
@wmat : I am not sure I am following: my PR removed the image files and introduced the wavedrom snippets into each of the instruction `.adoc`.

IIRC riscv-isa-manual has also switched to incorporating the instruction descriptions directly in the extension chapter rather in separate files, I may have to do this as well.

wmat commented:
I don't believe we've switched to including the wavedrom as snippets directly in the .adoc.  Was there a thread about this somewhere?  riscv-isa-manual still uses separate .adoc files.

nibrunieAtSi5 commented:
> I don't believe we've switched to including the wavedrom as snippets directly in the .adoc. Was there a thread about this somewhere? riscv-isa-manual still uses separate .adoc files.

Looking at the official ratified vector crypto spec for example:
https://github.com/riscv/riscv-isa-manual/blob/472a62018afbd14e24e18b2583546dfa61d6ba0c/src/vector-crypto.adoc#L1198-L1210

```
Encoding (Vector-Vector)::
[wavedrom, , svg]
....
{reg:[
{bits: 7, name: 'OP-VE'},
{bits: 5, name: 'vd'},
{bits: 3, name: 'OPMVV'},
{bits: 5, name: '00001'},
{bits: 5, name: 'vs2'},
{bits: 1, name: '1'},
{bits: 6, name: '101000'},
]}
....
```
This is what I could find, so I thought this was supposed to be the new way.

wmat commented:
I don't think we've ever landed on a preferred way tbh.  When I took over the doc conversions the method of including wavedrom as separate files was the method being used.  For the riscv-isa-manual specifically we are using bytefield-svg encoding of diagrams as well which I believe require using #include macros in the .adoc files.  Let me look into this more as I'd actually prefer not to use includes.

nibrunieAtSi5 commented:
deprecated in favor of the PR against **riscv-isa-manual**
